crates/swc_ecma_transforms_compat/src/es2015/classes/mod.rs

```diff
@@ -443,7 +454,7 @@ where
                     init: Some(Box::new(Expr::Call(CallExpr {
                         span: DUMMY_SP,
                         callee: helper!(create_super, "createSuper"),
-                        args: vec![class_name.clone().as_arg()],
+                        args: vec![Expr::Ident(class_name_sym).as_arg()],
```

Comment:
Please us `class_name_sym.as_arg()` instead of `Expr::Ident(class_name_sym).as_arg()`.